### ISSUE TITLE
Dead link

### ISSUE THREAD

**Issue by danadascalescu00** [pytorch/pytorch.github.io#1256]
## 📚 Documentation

Dead link found on: https://pytorch.org/tutorials/beginner/finetuning_torchvision_models_tutorial.html#load-data (_"... as described [here](https://pytorch.org/docs/master/torchvision/models.html)..."_) . The link should lead to the explanation of the hard-coded normalization values of the pretrained models on ImageNet.
